modules/programs/thunderbird.nix

```diff
@@ -49,6 +49,11 @@ let
       "mail.identity.id_${id}.fullName" = account.realName;
       "mail.identity.id_${id}.useremail" = address;
       "mail.identity.id_${id}.valid" = true;
+      "mail.identity.id_${id}.htmlSigText" =
+        if account.signature.showSignature == "none" then
+          ""
```

Comment:
are we sure this is the correct value and not something like `false` ? I couldn't find a schema of the config

Reply:
I was thinking of the thunderbird config

Reply:
Or do you mean the Thunderbird config `htmlSigText`? That's a string. It's what is inside the text field in the identity config window. And `""` makes that field empty.